### ISSUE TITLE
suggestion: when changing `path`, should `file` adjust accordingly (?)

### ISSUE THREAD

**Issue by LukasGentele** [jerray/publish-docker-action#13]
Currently, if you want to build a Dockerfile in a subdirectory, you'll have to adjust `path` and `file`. However, when changing the context using `docker build some/other/context`, Docker would also prefer the file `some/other/context/Dockerfile` according to the context (if not specified otherwise). 

Should this action also follow this style and use the Dockerfile within `path` if not otherwise specified? Would be a breaking change but I think it would be more consistent with the Docker CLI experience. 

@jerray Happy to submit a PR if you think this idea makes sense.

**Comment by jerray** [comment]
Good idea! I didn't think too much about relation between `path` and `file` when I create this action.

Looking forward to see your PR. Thank you!

**Comment by patsevanton** [comment]
Any news?
